app/frontend/packs/application.scss

```diff
@@ -1,4 +1,6 @@
-@charset "UTF-8";
+@import "../css/template.scss";
+
+// Application styles
```

Comment:
For an ongoing project you may also want to put these in like a `layout.scss` or something but since it's short it's fine to keep them here.